docs/source/data-access-api/index.rst

```diff
@@ -16,21 +16,201 @@ that the full implementation of the functionality is yet to be completed.*
+        # The following line is returned:
+        [{'file_path': 'imap/swe/l0/2024/01/imap_swe_l0_sci_20240105_v001.pkts', 'instrument': 'swe', 'data_level': 'l0', 'descriptor': 'sci', 'start_date': '20240105', 'version': 'v001', 'extension': 'pkts'}, {'file_path': 'imap/swe/l0/2024/01/imap_swe_l0_sci_20240105_v001.pkts', 'instrument': 'swe', 'data_level': 'l0', 'descriptor': 'sci', 'start_date': '20240105', 'version': 'v001', 'extension': 'pkts'}]
+
+Download
+^^^^^^^^
+
+To download files using the CLI tool, use the command ``download``. These will be placed in a ``data`` directory.
+
+It is important to note that your working directory will be established as the default directory. I.e, the ``data``
+directory will automatically be placed in this file path. Choose your working directory
+accordingly to suit your desires.
+
+When downloading a file from the API, different folders within the ``data`` directory will be made to better
+organize the downloaded files. See the example path: ``data/imap/swe/l0/2024/01/imap_swe_l0_sci_20240105_20240105_v00-01.pkts``
+
+**Example Usage:**
+
+    .. code-block:: bash
 
-When uploading files to the API, ensure these files are stored properly in a ``data`` directory. Then,
-ensure your working directory is one level above the ``data`` directory in order to properly upload files.
+        imap-data-access download imap/swe/l0/2024/01/imap_swe_l0_sci_20240105_v001.pkts
+
+
+Upload
+^^^^^^
+
+Similarly, files can be uploaded to the API using the command ``upload``.
+
+When uploading files to the API, ensure these files are stored properly in a ``data`` directory (see the Data Directory section below for more information). Then,
+ensure your working directory is one level above ``data`` in order to properly upload files.
 
 [WIP] Certain ancillary files can also be uploaded to the API. For more specific information regarding these files, visit
 `Ancillary Files <https://imap-processing.readthedocs.io/en/latest/data-access-api/calibration-files.html>`_
 
 **Example Usage:**
 
-.. code-block:: bash
+    .. code-block:: bash
+
+        imap-data-access upload /imap/swe/l1a/2024/01/imap_swe_l1a_sci_20240105_v001.cdf
+
+Importing as a package
+----------------------
+Imap data access can also be imported and used as a python package if desired.
+
+**Example Usage:**
+
+    .. code-block:: bash
+
+        import imap_data_access
+
+        # Search for files
+        results = imap_data_access.query(instrument="mag", data_level="l0")
+        # results is a list of dictionaries
+        # [{'file_path': 'imap/swe/l0/2024/01/imap_swe_l0_sci_20240105_v001.pkts', 'instrument': 'swe', 'data_level': 'l0', 'descriptor': 'sci', 'start_date': '20240105','version': 'v001', 'extension': 'pkts'}, {'file_path': 'imap/swe/l0/2024/01/imap_swe_l0_sci_20240105_v001.pkts', 'instrument': 'swe', 'data_level': 'l0', 'descriptor': 'sci', 'start_date': '20240105', 'version': 'v001', 'extension': 'pkts'}]
```

Comment:
Similar to my comment above, if this could be split into multiple lines I think that would read better